app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -2,7 +2,21 @@ class ApplicationHelper::Button::HostFeatureButton < ApplicationHelper::Button::
   needs :@record
 
   def visible?
-    return false if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
+    unless @feature.nil?
+      if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
+        return true if %w(start stop).include?(@feature.to_s)
+        return false
+      end
+
+      if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
+        case @feature.to_s
```

Comment:
@romanblanco Oh I didn't realize that was possible. That makes it much cleaner. Thank you.